src/mscorlib/ref/Compat/mscorlib.cs

```diff
@@ -2601,6 +2607,8 @@ public partial class Exception : System.Runtime.Serialization.ISerializable
         public virtual string StackTrace { get { return default(string); } }
         public System.Reflection.MethodBase TargetSite { get { return default(System.Reflection.MethodBase); } }
         protected event System.EventHandler<System.Runtime.Serialization.SafeSerializationEventArgs> SerializeObjectState { add { } remove { } }
+        [System.Runtime.CompilerServices.FriendAccessAllowedAttribute]
+        internal void AddExceptionDataForRestrictedErrorInfo(string restrictedError, string restrictedErrorReference, string restrictedCapabilitySid, object restrictedErrorObject, bool hasrestrictedLanguageErrorObject = false) { }
```

Comment:
@ericstj correct me if I'm wrong but I don't think we really have a way to express that in our nuget packages. Today we use tfm to indicate support and when we ship a new package we are supposed to be backwards compatible with older platforms. So doing what you suggest would be a breaking change. I do however agree that about our explosion of our test matrix and I think we need to figure out a strategy for doing this.